src/Components/Cards/SingleArticle.jsx

```diff
@@ -2,29 +2,33 @@ import { convertTimestampToDate } from "../../Utils/Utils"
 import { AuthorCard } from "./AuthorCard"
 import * as API from '../../Api'
 import { useState, useEffect } from "react"
+import { Voter } from "../Voter"
 export const SingleArticle = ({article}) => {
     const {title, topic, author, body, created_at, votes} = article
 
-    const [user, setUser] = useState(author)
+    const [user, setUser] = useState(null)
+    const [articleAuthor, setArticleAuthor] = useState(author);
```

Comment:
do you need both a user state and an author state?

Reply:
Ill look into that, the api get wasnt working without it

Reply:
Also I was getting errors in the author card when iI was setting state to just the username